src/main/java/com/ldtteam/structurize/network/NetworkChannel.java

```diff
@@ -60,7 +61,6 @@ public void registerCommonMessages()
         registerMessage(++idx, StructurizeStylesMessage.class, StructurizeStylesMessage::new);
         registerMessage(++idx, UndoMessage.class, UndoMessage::new);
         registerMessage(++idx, UpdateScanToolMessage.class, UpdateScanToolMessage::new);
-        registerMessage(++idx, UpdatePlaceholderBlockMessage.class, UpdatePlaceholderBlockMessage::new);
```

Comment:
Even though we usually don't allow running different structurize version at client and server, please keep the idx increment and add todo remove 1.17 comment

Reply:
Nah, we do this regularly in minecolonies. I don't have a problem with this.